server/src/main/java/org/opensearch/index/codec/composite/composite99/Composite99DocValuesReader.java

```diff
@@ -164,7 +164,7 @@ public Composite99DocValuesReader(DocValuesProducer producer, SegmentReadState r
 
                             // adding metric fields
                             for (Metric metric : starTreeMetadata.getMetrics()) {
-                                for (MetricStat metricStat : metric.getMetrics()) {
+                                for (MetricStat metricStat : metric.getBaseMetrics()) {
```

Comment:
how will SUM, MIN, MAX will be handled since they don't have any base metrics? It should be getMetrics only right?

Reply:
okay I see we are adding baseMetrics to metric